tools/stats-definition-exporter/README.md

```diff
@@ -0,0 +1,12 @@
+# Sync Gateway Prometheus Stat Definition Exporter
+This tool exports all the Prometheus Sync Gateway stats definitions to a JSON file. The JSON file is put inside of a gzipped tarball archive. The JSON is prettified by using tab indents and new lines.
+
+## Output
+### To File
+By default, this tool will output to a file. The archive file outputted is called `sync_gateway_metrics_metadata.tar.gz` however if the string `@PRODUCT_VERSION@` has been replaced in the `main.go` file, the archive file will be called  `sync_gateway_metrics_metadata_PRODUCT_VERSION.tar.gz`. The JSON file inside the archive is always called `sync_gateway_metrics_metadata.json`. This is outputted in the current working directory.
```

Comment:
Did you discuss this with Chris? For server the build team creates the compressed file for us... Additionally do we want the product version appended?

Reply:
For this, I was just going off the comment in the ticket: https://issues.couchbase.com/browse/CBG-3190?focusedId=694842&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-694842

Reply:
Worth double checking with @malarky. On server I'm 90% sure it was done on the build side.

Reply:
My opinion is to keep this tool only ever producing a single uncompressed fixed-name file and let the build team do whatever they want to it (compress it, name it with a version/build number, etc.)

It's more difficult to do this here than gzipping as part of a build step.

Reply:
Yeah, the file should be left uncompressed, and without a version number - let the build team archive it up 👍🏻